### ISSUE TITLE
Kabanero stuck on old openshift-knative operators

### ISSUE THREAD

**Issue by bbrowning** [kabanero-io/kabanero-operator#181]
The openshift-knative operator that you're consuming will no longer be published to quay.io as you've been consuming it. Instead, future versions will be published to registry.redhat.io and only available to users with valid pull secrets to the Red Hat Container Catalog.

Additionally, the next version explicitly requires OCP 4.1 and OpenShift Service Mesh 1.0. It will not work on OKD 3.11 nor with upstream Istio. There are no future releases planned to the quay.io/openshift-knative coordinates you're using. Our Knative Serving and Eventing Operators are also in the process of being combined into one operator as far as OLM is concerned.

In short, we should chat. Otherwise, Kabanero will be stuck on old images that are not supported and explicitly not meant to ever be used in production.

**Comment by kaczyns** [comment]
FWIW this appears to extend beyond just the operator.
https://github.com/knative/pkg/pull/683/

**Comment by bbrowning** [comment]
Yes - Knative 0.10 and newer will require Kubernetes 1.14 or newer. Technically it will likely only need Kubernetes 1.13.10/OpenShift 4.1 or newer (due to a bug fix with multiple API versions of CRDs). That's exactly the requirement our operator imposes as well, for this reason.

**Comment by kaczyns** [comment]
We are on a path now towards OpenShift 4.2 and the OpenShift Serverless operator.  I am going to close this issue.  @bbrowning thanks for your help as we make the transition!

**Comment by bbrowning** [comment]
Great! Feel free to ping me if you run into any more issues and I'll try to keep an eye on things to proactively point out anything I find.
